onnxruntime/python/tools/transformers/models/whisper/requirements.txt

```diff
@@ -7,7 +7,7 @@ soundfile
 librosa
 optimum
 onnxruntime-extensions>=0.9.0
-onnx>=1.16.0
+onnx>=1.15.0
```

Comment:
Log down the error messages, create ADO items for each error you found.

Reply:
Did you create the work item for linux tvm

Reply:
No.